src/Components/AppBarCollapse.js

```diff
@@ -30,16 +31,26 @@ const styles = (theme) => ({
 const AppBarCollapse = (props) => (
   <div className={props.classes.root}>
     <ButtonAppBarCollapse>
-      <Link to='/login'><MenuItem>Login</MenuItem> </Link>
-      <Link to='/signup'><MenuItem>Signup</MenuItem></Link>
+      <Link to="/login">
+        <MenuItem>Login</MenuItem>{' '}
```

Comment:
```suggestion
        <MenuItem>Login</MenuItem>{' '}
```
do we need the  " " space? can we just use css?